src/components/Button.tsx

```diff
@@ -0,0 +1,14 @@
+export const primary = 'bg-blue text-white hover:bg-Web-dev';
+export const secondary = 'text-blue border border-2 border-blue hover:text-white hover:bg-blue';
+
+export const Button = ({ variant, text }: { variant: string; text: string }) => {
+  return (
+    <button
+      className={`m-8 flex h-10 w-[104px] flex-row items-center justify-center rounded text-[16px] font-bold leading-[19px] hover:shadow-button ${variant}`}
```

Comment:
I would add padding so it will be closer to the design. The secondary button looks a bit off.

Reply:
Done. Thanks for the comment.